math/dec.go

```diff
@@ -99,8 +138,19 @@ func (x Dec) Sub(y Dec) (Dec, error) {
 	return z, errors.Wrap(err, "decimal subtraction error")
 }
 
-// Quo returns a new Dec with value `x/y` (formatted as decimal128, 34 digit precision) without mutating any
-// argument and error if there is an overflow.
+// Quo performs division of x by y using the decimal128 context with 34 digits of precision.
```

Comment:
here `decimal128` is correct for the rounding. It would be helpful to mention the rounding rules as well.